tools/m1_utils/update_static.sh

```diff
@@ -10,29 +10,22 @@ unarchive() {
 
     rm -rf *.o 
     ar -x "$FILE" 
-    ar t "$FILE" | grep \.o$ | sort > objs.txt
+    ar t "$FILE" | grep \.o$ | sort --ignore-case > objs.txt
 
     # Assume default extraction strategy is OK if no dupes
-    if [[ -z "$(uniq -d objs.txt)" ]]; then
+    if [[ -z "$(uniq -di objs.txt)" ]]; then
        return 0
     fi
 
+    uniq -D -i objs.txt > duplicate_objs.txt
```

Comment:
This `uniq` is duplicative of the one above? Consider: writing `duplicate_objs.txt` prior to the empty check, and checking the size of the file to exit early. After making this change, it looks like the `uniq` call might fit into the first pipeline:
```
    ar t "$FILE" | grep \.o$ | sort --ignore-case | uniq -D -i > duplicate_objs.txt
```